docs/data-sources/cbh_availability_zones.md

```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "Cloud Bastion Host (CBH)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_cbh_availability_zones"
+description: |-
+  Use this data source to get the list of CBH availability zones within HuaweiCloud.
+---
+
+# huaweicloud_cbh_availability_zones
+
+Use this data source to get the list of CBH availability zones within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "availability_zone_id" {}
+
+data "huaweicloud_cbh_availability_zones" "test" {
+  availability_zone_id = var.availability_zone_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the CBH availability zones.
+  If omitted, the provider-level region will be used.
+
+* `availability_zone_id` - (Optional, String) Specifies the ID of the availability zone to be queried.
```

Comment:
what is the ID
```suggestion
* `name` - (Optional, String) Specifies the name of the availability zone to be queried.
```
refer to the description of the cbh instance parameter `availability_zones`

Reply:
done.